tools/topology/topology1/sof-adl-es83x6-ssp1-hdmi-ssp02.m4

```diff
@@ -0,0 +1,256 @@
+#
+# Topology for generic Alderlake-P with es8316 codec, lt6911's.
```

Comment:
no, that's too much duplication and it's unmaintainable.

I would recommend using the same sof-glk-es8336 and conditionally include an 'hdmi-in.m4' in which you can add your own stuff.